TestAssets/TestProjects/EndToEndTestApp/project.json

```diff
@@ -20,11 +20,12 @@
     "files": {
       "includeFiles": "packfiles/pack1.txt",
       "mappings": {
-        "newpath/": "packfiles/pack2.txt"
+        "newpath/": "packfiles/pack2.txt",
+        "anotherpath/": "packfiles/pack2.txt"
       }
     }
   },
-  "publishOptions": "testpublishfile.txt",
+  "publishOptions": [ "testpublishfile.txt", "publishfiles" ],
```

Comment:
We have coverage for that scenario in `embed` and `packOptions`. Since all of these go through the same code paths, I don't think adding another test just for this scenario makes sense.